README.md

```diff
@@ -2,7 +2,7 @@
 
 centralized-sequencer is an implementation of the [Generic Sequencer interface](https://github.com/rollkit/go-sequencing)
 for modular blockchains. It runs a gRPC service,
-which can be used by rollup clients to sequence transactions to Celestia da.
```

Comment:
@gupadhyaya I'm assuming this was out of habit of testing on celestia, but this is just implemented against the go-DA interface correct? So it is DA layer agnostic?

Reply:
yes correct.